packages/components/src/playground/button.tsx

```diff
@@ -1,29 +1,34 @@
 import React from "react";
+import Typography from "../typography/typography";
 import tw from "twin.macro";
 
 // Temporary props for button to test out css options for variants
 type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
   variant?: "primary" | "secondary";
 };
 
-const ButtonComponent = tw.button`
-  font-bold py-2 px-4 rounded text-white border-none
+const StyledButton = tw.button`
+  py-2 px-4 rounded border-none
 `;
 
-const PrimaryButton = tw(ButtonComponent)`
+const PrimaryButton = tw(StyledButton)`
   bg-brand-600 hover:bg-brand-700
 `;
-const SecondaryButton = tw(ButtonComponent)`
+const SecondaryButton = tw(StyledButton)`
   bg-neutral-400 hover:bg-neutral-500
 `;
 
 function Button(props: ButtonProps): JSX.Element {
-  const { variant = "primary", ...rest } = props;
+  const { children, variant = "primary", ...rest } = props;
+  const ButtonComponent =
+    variant === "primary" ? PrimaryButton : SecondaryButton;
 
-  return variant === "primary" ? (
-    <PrimaryButton {...rest} />
-  ) : (
-    <SecondaryButton {...rest} />
+  return (
+    <ButtonComponent {...rest}>
+      <Typography bold color="white" size="button">
```

Comment:
wasn't 100% sure why there was a `Button` size in Figma, but I'm guessing this is how we'd use it? also fine with reverting this if that's not standard